jest.config.js

```diff
@@ -30,7 +30,11 @@ module.exports = {
       statements: 100,
     },
   },
-  moduleNameMapper: {'\\.css$': 'identity-obj-proxy'},
+  moduleNameMapper: {
+    '\\.css$': 'identity-obj-proxy',
+    '^@testing-library/user-event$': '<rootDir>/tools/test/act-user-event.ts',
```

Comment:
question: why these changes? What are we trying to solve

Reply:
Looks like this is a monkey patch. Let's add a comment with context for future maintainers please.

Include this link: https://github.com/adobe/react-spectrum/blob/main/packages/dev/test-utils/src/renderOverride.js#L21-L29

Reply:
Yes! Not a problem, I'll add comments with the link included